### PR TITLE
Plone 5.1 compatibility

### PR DESCRIPTION
**Pull request by djowett-ftw**
Close https://github.com/4teamwork/izug.organisation/issues/1446

This passes tests, but:

- I'm not sure about the publisher work around
- the control panel displays the status message twice for Plone 5 (or 3 times if using the blueberry theme)

Screenshot 2019-09-06 at 16 38 50

### CODE REVIEW

File: ftw/globalstatusmessage/tests/test_controlpanel.py [hunk 1]
```diff
@@ -140,7 +147,8 @@ def test_receiver(self, browser):
         browser.login().open(
             self.portal,
             view='@@global_statusmessage_config_receiver',
-            data={'jsondata': self.payload})
+            data={'jsondata': self.payload},
+            send_authenticator=HAS_MODERN_PLONE_PROTECT)
```
Comment (jone) [inline ftw/globalstatusmessage/tests/test_controlpanel.py]: Did you run into an import error on Plone 4 in `ftw.testbrowser` when setting this always to `True`?

I'd prefer that we can set `send_authenticator=True` and `ftw.testbrowser` would detect wether it is necessary or not 🤔

**Comment by jone** [comment]
**Publisher:**
The publisher part is hard to test because we do not have an end-to-end test-setup for the publisher. But since there are no changes on the storage of the messages at all it should keep working. I've made a note that we will test this manually in QA.

**Duplicate messages:**
Could you find the reason for duplicate messages already?
I'd first test whether there are duplicates without diazo.
If not, I suspect that a diazo rule in the theme duplicates the messages somehow.

**Comment by djowett-ftw** [comment]
> **Publisher:**

Ok - I misunderstood that part as we found in our useful corridor discussion! No problem then.

> 
> **Duplicate messages:**
> Could you find the reason for duplicate messages already?
> I'd first test whether there are duplicates without diazo.
> If not, I suspect that a diazo rule in the theme duplicates the messages somehow.

Diazo is doubling the message at the top; yet to uncover how, is that a fix for now, or should I add an issue to the blueberry theme?
The message in the middle of the screen is still there with Diazo off.  Does that matter?

**Comment by jone** [comment]
> Diazo is doubling the message at the top; yet to uncover how, is that a fix for now, or should I add an issue to the blueberry theme?

Please add an issue; we can't fix it here.

> The message in the middle of the screen is still there with Diazo off. Does that matter?

I think that was not there before and we should find the cause and remove the message in the middle.

**Comment by djowett-ftw** [comment]
The duplicate global status messages are a Plone issue I believe.  I've raised it here https://github.com/plone/Products.CMFPlone/issues/2947

Worked around it using CSS in cde161c.

**Comment by djowett-ftw** [comment]
Resolves https://github.com/4teamwork/izug.organisation/issues/1446

**Pull request merged**